app/components/T/tests/index.test.tsx

```diff
@@ -24,21 +26,25 @@ describe('<T /> component tests', () => {
     expect(getAllByText(/List of launches/).length).toBe(1);
   });
 
-  it('should have a margin-bottom of 5px when we pass marginBottom as 5', () => {
+  it('should have a margin-bottom of 5rem when we pass marginBottom as number', () => {
     const props = {
       marginBottom: 5,
       id: 'launches_list'
     };
-    const styles = getComponentStyles(T, props);
-    expect(styles.marginBottom).toBe(`${props.marginBottom}rem`);
+
+    const { getByTestId } = renderWithIntl(<T {...props} />);
+    const styledText = getByTestId('t');
+    expect(styledText).toHaveStyle({ marginBottom: '5rem' });
   });
```

Comment:
check the tests. Why is it changed ?

Reply:
getComponentsStyles is removed, it was being used for styled-components